### PR TITLE
test(pie-tag): DSW-1520 update inline icon to webc icon in percy test

### PR DESCRIPTION
**Pull request by dandel10n**
- components updated: pie-link, pie-button, pie-icon-button, pie-tag
- removes inlined icon
- mounts tag and icon components in visual test

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] If it is a component change, I have reviewed the Storybook preview
- [x] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [x] If there are visual test updates, I have reviewed them

### Reviewer 2
- [x] If there are visual test updates, I have reviewed them

### CODE REVIEW

**Review by ashleynolan — DISMISSED** [review]

**Review by raoufswe — APPROVED** [review]

**Review by fernandofranca — DISMISSED** [review]

**Review by jamieomaguire — DISMISSED** [review]

File: packages/components/pie-icon-button/test/visual/pie-icon-button.spec.ts [hunk 1]
```diff
@@ -24,32 +25,23 @@ const props: PropObject = {
...
 const componentVariants: string[] = Object.keys(componentPropsMatrixByVariant);
 
-// This ensures the component is registered in the DOM for each test
-// This is not required if your tests mount the web component directly in the tests
-test.beforeEach(async ({ page, mount }) => {
-    await mount(
-        PieIconButton,
-        {},
-    );
+// eslint-disable-next-line no-empty-pattern
```
Comment (xander-marjoram) [inline packages/components/pie-icon-button/test/visual/pie-icon-button.spec.ts:35]: What is causing this linting error? It doesn't look like it should be necessary

File: packages/components/pie-tag/test/visual/pie-tag.spec.ts [hunk 2]
```diff
@@ -37,8 +32,15 @@ const componentPropsMatrixByVariant: Record<string, WebComponentPropValues[]> =
 const componentVariants: string[] = Object.keys(componentPropsMatrixByVariant);
 
 // eslint-disable-next-line no-empty-pattern
-test.beforeEach(async ({ }, testInfo) => {
+test.beforeEach(async ({ mount }, testInfo) => {
```
Comment (xander-marjoram) [inline packages/components/pie-tag/test/visual/pie-tag.spec.ts:35]: And the line above here, too

**Review by xander-marjoram — COMMENTED** [review]
Lovely stuff, this has needed doing for a while!

File: packages/components/pie-icon-button/test/visual/pie-icon-button.spec.ts (hunk 1 above)
Comment (dandel10n) [reply packages/components/pie-icon-button/test/visual/pie-icon-button.spec.ts]: Oh, good catch, there used to be an empty object but after a few changes there is a mount in it now so there is no need for the ignore 👍

**Review by fernandofranca — APPROVED** [review]

**Review by raoufswe — APPROVED** [review]

**Pull request merged**